### DISCUSSION TITLE
Deploy -> Kubernetes -> AdminUtils

### DISCUSSION THREAD

**Discussion by monojit18** [Sunbird-Ed/Community#156] [Installation] [answered]
Deploy job for AdminUtils failing and the corresponding shows the below log.
Branch: release-4.10.1_RC1

**Note**: _Keycloak_ run is successful and Realm also created. All the secrets variable are updated as per instructions in the common.yaml file.

Can you please help/suggest?

=====================================================================================

ERROR o.s.boot.SpringApplication.reportFailure - Application startup failed
org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'tokenSignStepChain': Unsatisfied dependency expressed through field 'keyManager'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'keyManager': Invocation of init method failed; nested exception is java.security.spec.InvalidKeySpecException: java.security.InvalidKeyException:

=========================================================================================

**Comment by beepdot** [comment, marked as answer]
Can you share the adminutils configmap data please?

**Reply by monojit18** [reply]
I am guessing you are referring to this one:
/var/lib/jenkins/workspace/Deploy/dev/Kubernetes/AdminUtils/deploy/config.yml?
PFA
[config-adminutil.txt](https://github.com/Sunbird-Ed/Community/files/9558278/config-adminutil.txt)

Please let me know if this helps?

**Reply by monojit18** [reply]
@keshavprasadms does this info help?

**Reply by beepdot** [reply]
Deploy the adminutils job and when the job is deploying and running `helm install` sections, you will be able to see a configmap named `adminutils-config`. Please share me the kuberenets configmap.

**Reply by monojit18** [reply]
@keshavprasadms This is now working. I had to helm uninstall the earlier AdminUtils release and then run the job again. Thanks for the help...marking it as answered.
